vm/block_test.go

```diff
@@ -0,0 +1,49 @@
+package vm
+
+import "testing"
+
+func TestBlockInitialize(t *testing.T) {
+	tests := []struct {
+		input    string
+		expected interface{}
+	}{
+		{`
+b = Block.new do
+  100
+end
+
+b.call
+`, 100},
+
+		{`
```

Comment:
@saveriomiroddi @hachi8833 Do you think we need more complex cases like this?

Reply:
Looks sufficient by now.